rest-service/manager_rest/test/endpoints/test_blueprints.py

```diff
@@ -461,7 +462,6 @@ def test_blueprint_update_invalid_param_type(self):
     @mock.patch('requests.get', side_effect=mocked_requests_get)
     def test_blueprint_upload_autoupload_plugins_network_failure(
             self, mock_get, mock_get_managers):
-        mock_get.side_effect = Exception('Faulty network')
```

Comment:
doesn't this test need to have some exception in it :P

Reply:
I use requests rather than self._download_file now for getting the catalog, so I don't need to raise an exception to simulate network failure, just a 400 response

Reply:
aha, 404 in this case, `mocked_requests_get`